pkg/apis/network/v1/pki_types.go

```diff
@@ -0,0 +1,60 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PKISpec defines the desired state of PKI.
+// +k8s:openapi-gen=true
+type PKISpec struct {
+	// caCert is information about the CA certificate and key.
+	CACert CertSpec `json:"caCert"`
```

Comment:
Is this information that the user provides or information that the controller fills in? If the former, what gets done with that information?